### ISSUE TITLE
USER STORY: Model Training

### ISSUE THREAD

**Issue by Hasmelash95** [Hasmelash95/breast-cancer-detector#16]
As a **Developer** I can **use the training dataset** so that **I can train a model to accurately predict the class of an image dataset**.

**Acceptance Criteria**
* Create a tensor flow model with convolution, pooling and dense layers. 
* Add a dropout layer to reduce overfitting.
